vpr/src/base/read_options.cpp

```diff
@@ -1754,6 +1785,97 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .default_value("false")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    place_grp.add_argument(args.place_static_move_prob, "--place_static_move_prob")
+        .help(
+            "The Probability of different move in Simulated Annealing."
```

Comment:
Please make this more explicit.  "The percentage probabilities of different moves in Simulated Annealing placement. The values should all be between 0 and 100, and should sum to 100.  The numbers listed are interpreted as the percentage probabilities of {uniformMove <insert rest>}, in that order."

Reply:
Updated! However, I have removed the constraint that they should sum up to 100. I just care about the relative percentages.